numalogic/models/threshold/_static.py

```diff
@@ -68,45 +67,58 @@ def score_samples(self, x: npt.NDArray[float]) -> npt.NDArray[float]:
 
 class SigmoidThreshold(BaseThresholdModel):
     r"""Smooth and stateless static thesholding using sigmoid function as an estimator.
-    The values produced.
 
     Score is given by:
-            score = score_limit * 1/ exp(-coeff * (x - upper_limit))
+            score = score_limit * 1/ exp(-coeff * (x - upper_limits))
```

Comment:
Should we save the list of scores for each upper_limit in the list of floats
meaning, we would have score for each feature depending on number of upper_limits in the list

Reply:
But we receive `x` at runtime, which would determine the scores.